docs/index.md

```diff
@@ -0,0 +1,125 @@
+# Pay with Card
+
+Accept card payments in your app using the PayPal In-App Payments SDK.
+
+## How it works
+
+The Payments SDK allows for merchant apps to integrate easily with PayPal payment services on both the client and server.
+
+You will need a server integration to create an order and capture the funds using [PayPal Orders v2 API](https://developer.paypal.com/docs/api/orders/v2). The Payments SDK allows for client-side approval of an order with the user's card information.
+
+## Eligibility
+```
+# Placeholder: Product Eligibility
+what kind of merchants are we targetting?
+what features do we offer right now/what features are we missing?
+```
+
+## Integration methods
+
+- [Card Fields UI Integration](#card-fields-ui-integration)
+- [Fully Customizable Integration](#fully-customizable-integration)
+
+## Card Fields UI Integration
+
+Accept card payments using our card fields UI
+
+```
+# Placeholder: Include images of card fields
+```
+
+**Requirements:**
+Card fields UI offers a low effort integration where you can use our card fields UI, and our card fields will handle ...
```

Comment:
I feel like we should move away from low effort vs high effort language as this could vary by dev experience, how their app set up, and potentially cause frustration with our SDKs. Maybe low vs high customization would be better? or something else? Thoughts?

Reply:
I've seen no-code and low-code, but for traditional development I'm not sure of the marketing term lol. We could go with "drop-in" here since that's technically its purpose, but I'm up for making the language read more like an executive summary. The person evaluating our SDK may not always be the integration engineer.

Reply:
Hahaha, yeah maybe this is something marketing can help us with.